### ISSUE TITLE
Running Pipeline Command Issue

### ISSUE THREAD

**Issue by kyoobin10** [allenday/dataflow-genomics#30]
java -cp target/giab-example-1.0.0.jar \
    com.google.allenday.popgen.PopGenProcessingApp \

Shouldn't the directory be different? 
Not PopGenProcessingApp but GiabExampleApp?

**Comment by pseveryn** [comment]
Yes, sure. Thank you! Fixed in: https://github.com/allenday/dataflow-genomics/pull/31
